src/base64.cr

```diff
@@ -96,39 +68,13 @@ module Base64
   # Tm93IGlzIHRoZSB0aW1lIGZvciBhbGwgZ29vZCBjb2RlcnMKdG8gbGVhcm4gQ3J5c3RhbA==
   # ```
   def strict_encode(data) : String
-    strict_encode data, CHARS_STD, pad: true
-  end
-
-  private def strict_encode(data, alphabet, pad = false)
     slice = data.to_slice
     String.new(encode_size(slice.size)) do |buf|
-      appender = buf.appender
-      to_base64(slice, alphabet, pad: pad) { |byte| appender << byte }
-      size = appender.size
-      {size, size}
+      written_bytes = encode_base64_buffer_internal(slice.to_unsafe, slice.size, buf, CHARS_STD.to_unsafe, pad: true)
+      {written_bytes, written_bytes}
```

Comment:
Nitpick, but the variable name here is not consistent with the way it's written in the other methods

```suggestion
      bytes_written = encode_base64_buffer_internal(slice.to_unsafe, slice.size, buf, CHARS_STD.to_unsafe, pad: true)
      {bytes_written, bytes_written}
```